src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,6 +10,7 @@
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_ID = new Prefix("i/");
     public static final Prefix PREFIX_SERVICEID = new Prefix("si/");
+    public static final Prefix PREFIX_BOOKINGID = new Prefix("bi/");
```

Comment:
Prefix does not appear anywhere else in the PR, unsure what's the reason for adding?